config/crd/bases/misf.ibm.com_managedfusionofferings.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: managedfusionofferings.misf.ibm.com
+spec:
+  group: misf.ibm.com
```

Comment:
just a query, why `misf`

Reply:
It is according to the design, even I was not able to figure out the full form of misf. I will add a comment on the design doc.

Reply:
I remember reading most of ibm fusion lineup are part of this group, don't know the full form though 😅

Reply:
Full form of misf: Managed IBM Storage Fusion

Reply:
@leelavg `isf` original meaning was `IBM Spectrum Fusion` which is now `IBM Storage Fusion` (have the same abbr), this makes the managed flavor either `isf` or `misf`. After discussion with the Fusion owners they requested differentiation which resulted at as choosing `misf`